microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -36,6 +40,13 @@ This release adds improvements related to the following components and concepts.
 ** If you run cAdvisor as a standalone DaemonSet for monitoring pods and containers, update it to v0.43.0 or later.
 ** If you deploy Java applications with the JDK, ensure you are using JDK 11.0.16 and later or JDK 15 and later, which fully support cgroup v2.
 
+[id="microshift-4-14-updates-supported"]
+=== Updates are supported on {product-version} and later
+With this release, both minor-version and z-stream releases updates are supported on both {op-system-ostree} edge-deployed hosts and non-OSTree hosts.
```

Comment:
We are mixing language here - I am not sure everybody understands what z-stream means. We are using semantic versioning, which is either major.minor.patch, or X.Y.Z versioning. So I suggest we either speak of minor- and patch- updates, or Y-stream and Z-stream updates. I would prefer the earlier one.

Reply:
I noticed this in other PRs, but checked OCP docs and it mixes `minor version` and `z-stream` as well: https://docs.openshift.com/container-platform/4.13/updating/understanding-upgrade-channels-release.html

Reply:
@pmtk The Lifecycle articles call the 4.x.y versions the "micro" versions. So it gets even more complicated.  https://access.redhat.com/support/policy/updates/openshift#ocp4_phases 
Let's go with major.minor.patch Per Daniel's suggestion for our purposes. I'll clean those up as we go along. If we must refer to a specific z-stream version at a later date, we can just list it as that (4.x.y) without calling it anything.

Reply:
btw, heres a link to a definition: https://semver.org/